mmaction/core/evaluation/eval_hooks.py

```diff
@@ -11,14 +12,20 @@
 class EvalHook(Hook):
     """Non-Distributed evaluation hook.
 
+    Notes:
+        If new arguments are added for EvalHook, tools/test.py,
+        tools/eval_metric.py may be effected.
+
     This hook will regularly perform evaluation in a given interval when
     performing in non-distributed environment.
 
     Args:
         dataloader (DataLoader): A PyTorch dataloader.
+        start (int, optional): Evaluation starting epoch. It enables evaluation
```

Comment:
start_epoch/starting_epoch? what if it is not epoch based?

Reply:
Truly the `EvalHook` perform evaluation using a cycle based on epoch. If it is not epoch based, like iter, users may need to customize a `IterEvalHook` or we provide a PR for it in the future when we need it.

Reply:
ok, rename it

Reply:
better. 

actually this is a sweet mistake. i meant `start` -> `start_epoch`

Reply:
Since I rename the class name and there is no ambiguity, there is no need to rename the `start` argument. And `start` is better for users or us to add `IterEvalHook` which may be a subclass for `EpochEvalHook`

Reply:
https://github.com/open-mmlab/mmediting/blob/master/mmedit/core/evaluation/eval_hooks.py

Reply:
fine